xbmc/video/tags/VideoInfoTagLoaderFactory.cpp

```diff
@@ -22,10 +22,6 @@ IVideoInfoTagLoader* CVideoInfoTagLoaderFactory::CreateLoader(const CFileItem& i
                                                               bool lookInFolder,
                                                               bool forceRefresh)
 {
-  // don't try to read tags for streams
-  if (item.IsInternetStream())
-    return nullptr;
-
```

Comment:
@notspiff you directly programmed it like this. Removing it without discussion sounds not right.

Reply:
Alternatively we could move the NFO-reading up so it's before the IsInternetStream()-check, although I don't see why any of the stuff below it could be a problem for http/https.

Reply:
in principle there is no problem loading for http(s). but some servers will not like the Exists() and some for some streaming protocols it makes little sense. so i added it to lessen chance of issues and log spam. i don't see a big issue removing it as such, but that was the logic.

Reply:
Yes, I already suspected that. Since we've ran into this issue before I think we should see whether we can find another way to flag/distinguish between actual "streams" and "files" served from http/https but that's beyond the scope of this PR.

Reply:
Also be aware of shoutcasts (internet radio). I don't know if they would get to this part of code, but I know that they are often over looked and we had issues with determining it was a shoutcast outside the player (the player knows but does not publish the fact)

Reply:
If somebody tries to scan a shoutcast station into their video library they deserve it all.